drivers/spi/spi_bitbang.c

```diff
@@ -551,4 +553,12 @@ FAR struct spi_dev_s *spi_create_bitbang(FAR const struct
   return &priv->dev;
 }
 
+void spi_destroy_bitbang(FAR struct spi_dev_s *dev)
+{
+  if (dev)
```

Comment:
Shouldn't this be a DEBUGASSERT(dev);?

Reply:
@xiaoxiang781216 why you resolved unanswered comment?

Reply:
Sorry, mistake.

Reply:
@pkarashchenko Fixed it to DEBUGASSERT.